views.py

```diff
@@ -347,3 +349,35 @@ def post(self, request):
                 'Unidentified Error',
                 status=status.HTTP_500_INTERNAL_SERVER_ERROR,
             )
+
+
+class CSVImportExport(APIView):
+    """
+    API endpoint that allows importing and exporting csv files
+    """
+
+    def get(self, request, *args, **kwargs):
+        """
+        Returns a blank xlsx file for given model
+        :return: a blank xlsx file for given model
+        """
+
+        model_name = request.GET.get('model')
+        if model_name is None or model_name not in models:
+            return Response(
+                { 'Invalid model': 'Model doesn\'t exists' },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+        Model = models[model_name]
+
+        response = HttpResponse(content_type='text/csv')
```

Comment:
What happens when you add the `content_type` for spreadsheet?

Reply:
Other alternative is `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`, by which the file is saved as an `Excel Worksheet` instead of a `CSV Document`. I'll check if we can parse worksheet using inbuilt libraries, if yes I'll later incorporate this change :thumbsup: